packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -387,6 +393,7 @@ function Cell(props) {
   const columnHeaderId = `${tableId}-${columnId}-headerCell`;
   const rowHeaderId = !isRowHeader && rowHeaderIndex !== -1 ? `${tableId}-rowheader-${rowId} ` : '';
   const sectionHeaderId = sectionId ? `${tableId}-${sectionId} ` : '';
+  const subsectionHeaderId = subsectionId ? `${tableId}-${sectionId}-${subsectionId} ` : '';
```

Comment:
Do we need the section id in the id for the subsection?

Reply:
I added it so we'll know which section the subsection is associated with. Do we need that or is the section id redundant?

Reply:
I guess having the section helps to ensure uniqueness.